File: W6/echon.py
#!/usr/bin/python2.7

from __future__ import print_function;
import sys;

if len(sys.argv) != 3:
   print ("Usage: ./echon.py <number of lines> <string>\n", end = '');
else :
   print ((sys.argv[2]+"\n") * int(sys.argv[1]), end = '');

File: W5/echon.pl
#!/usr/bin/perl -w

if (scalar @ARGV != 2) {
    print "Usage: ./echon.pl <number of lines> <string>\n";
}else{ 
    for ($n = $ARGV[0]; $n != 0; $n--) {  
        print "$ARGV[1]\n"; 
    }        
}
